MAUI/DataGrid/sorting.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: post
+title: Sorting in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Sorting support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
+platform: MAUI
+control: SfDataGrid
+documentation: UG
+---
+
+# Sorting in MAUI DataGrid (SfDataGrid)
+
+The [SfDataGrid](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html) provides the built-in support to sort one or more columns by using the [SfDataGrid.SortingMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SortingMode) property. When sorting is applied, the data grid automatically rearranges the data to match the current sort criteria. You can sort the data simply by tapping the column header. Once sorting is applied, the data grid shows a sort icon in the respective column header indicating the sort direction.
+
+## Programmatic sorting
+
+You can sort the data programmatically by adding or removing the [SortColumnDescription](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SortColumnDescription.html) in [SfDataGrid.SortColumnDescriptions](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_SortColumnDescriptions) property.
+
+The `SortColumnDescription` object holds the following two properties,
+
+* **ColumnName**: The name of a column to be sorted
+* **SortDirection**: An object of type `ListSortDirection` defines the sorting direction.
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="sfDataGrid" ItemsSource="{Binding OrderInfoCollection}" >
+
+    <syncfusion:SfDataGrid.SortColumnDescriptions>
+        <syncfusion:SortColumnDescription ColumnName="OrderID" SortDirection="Ascending" />
+    </syncfusion:SfDataGrid.SortColumnDescriptions>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+{% endtabs %}
+
+![Programmatical sorting .NET MAUI DataGrid](Images\sorting\maui-datagrid-programmatic-sorting.png)
+
+## Different modes of sorting
+
+`SfDataGrid` sorts the data against one or more columns based on the `SfDataGrid.SortingMode` property. The `SortingMode` contains the following properties,

```

Comment:
The `SfDataGrid` sorts the data against one or more columns based on the `SfDataGrid.SortingMode` property. The `SortingMode` contains the following properties:

Reply:
Modified